### PR TITLE
changes

### PR DESCRIPTION
**Pull request by mlzummo**
@GordonSmith Please Review
Signed-off-by: Mathew Zummo <mzummo@gmail.com>

### CODE REVIEW

File: src/form/Form.js [hunk 1]
```diff
@@ -19,15 +19,18 @@
             .columns([{
                 id: "subject",
                 label: "Subject",
-                type: "string"  //  If ommitted should default to string  ---
+                type: "string",  //  If ommitted should default to string  ---
+                validate: "[A-Za-z]"
```
Comment (GordonSmith) [inline src/form/Form.js]: "123!" is a valid string...

File: src/form/Form.js [hunk 2]
```diff
@@ -75,6 +82,24 @@
                  context.click();
              })
         ;
+
+    };
+
+    Form.prototype._validate = function(context) {
+        return this.columns().every(function(d, i) {
+            var valStr = d.validate;
```
Comment (GordonSmith) [inline src/form/Form.js]: What if validate is undefined?

File: src/form/Form.js (hunk 1 above)
Comment (mlzummo) [inline src/form/Form.js]: yea that's becuase regex still matched it .. gotta use something like ^[A-z]+$ ... this part isnt final and not even sure if we will include ... J has a refactor as well I need to look at

File: src/form/Form.js (hunk 2 above)
Comment (mlzummo) [inline src/form/Form.js]: (I need to test that i dont need a "typeof" check) but it should just return true .. that was the goal atleast but i will relook over before any real PR

**Pull request closed without merging**